packages/beacon-node/src/util/graffiti.ts

```diff
@@ -1,8 +1,37 @@
 import {GRAFFITI_SIZE} from "../constants/index.js";
+import {ExecutionEngineHttpOpts} from "../execution/engine/http.js";
+import {ClientCode, ClientVersion} from "../execution/index.js";
 
 /**
  * Parses a graffiti UTF8 string and returns a 32 bytes buffer right padded with zeros
  */
 export function toGraffitiBuffer(graffiti: string): Buffer {
   return Buffer.concat([Buffer.from(graffiti, "utf8"), Buffer.alloc(GRAFFITI_SIZE, 0)], GRAFFITI_SIZE);
 }
+
+export function getLodestarClientVersion(info?: ExecutionEngineHttpOpts): ClientVersion {
+  return {
+    code: ClientCode.LS,
+    name: "Lodestar",
+    version: info?.version ?? "",
+    commit: info?.commit?.slice(0, 4) ?? "",
+  };
+}
+
+export function getDefaultGraffiti(opts: {private?: boolean}, executionClientVersion?: ClientVersion): string {
```

Comment:
we need unit tests for this, I think it does not work as intended, see comment below